packages/react/ds/src/card/card.tsx

```diff
@@ -42,24 +67,43 @@ export const Card = (props: CardProps) => {
           </a>
         </div>
       )}
-      <div className="gi-card-content">
-        <div className="gi-card-heading">
-          <Heading as="h5" customClasses="!gi-my-0">
-            {href ? <Link href={href}>{title}</Link> : title}
-          </Heading>
+      <div className={`gi-card-content ${getInsetClass()}`}>
+        <div className="gi-card-header">
+          <div className="gi-card-heading">
+            <Heading
+              as="h5"
+              customClasses={`!gi-my-0 ${subTitle ? '!gi-mb-2' : ''}`}
+            >
+              {href ? <Link href={href}>{title}</Link> : title}
+            </Heading>
+            <Paragraph
```

Comment:
Don't use the Paragraph here. You can use the same classes or have one dedicated for the card subheading